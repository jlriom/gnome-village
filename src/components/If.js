import React from "react";
import PropTypes from "prop-types";
const If = props => (props.condition ? <>{props.children} </> : <> </>);

If.propTypes = {
	props: PropTypes.object
};

export default If;
