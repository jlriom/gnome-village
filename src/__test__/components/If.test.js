import React from "react";
import If from "../../components/If";
import renderer from "react-test-renderer";

const props = {
    condition: false
}

describe("<If />", () => {
    it("renders ok", () => {
        const tree = renderer.create(
            <If {...props} />
        );
        expect(tree).toMatchSnapshot();
    });
});
