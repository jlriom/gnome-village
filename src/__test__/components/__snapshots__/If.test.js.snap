// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<If /> renders ok 1`] = `" "`;
